docs/topics/auth/default.txt

```diff
@@ -672,7 +672,7 @@ email in the desired domain and if not, redirects to the login page::
 
     def my_view(request):
         if not request.user.email.endswith("@example.com"):
-            return redirect("/login/?next=%s" % request.path)
+            return redirect_to_login(request.get_full_path())
```

Comment:
`redirect_to_login()` is introduced later and this section refers to a "raw way", so I think it's fine to leave `redirect()` in this example.